git/README.md

```diff
@@ -91,28 +105,38 @@ When satisfied, they comment on the pull request `Ready to merge.`
 Rebase interactively. Squash commits like "Fix whitespace" into one or a small number of valuable commit(s). Edit commit
 messages to reveal intent. Run tests.
 
-    git fetch origin
-    git rebase -i origin/master
+```console
+git fetch origin
+git rebase -i origin/master
```

Comment:
`main` here, as well. Probably best to do a find/replace.

Reply:
I did a pass across all the guides 😉